docs/public/content/vuln_db/manuals/kb_importer.md

```diff
@@ -0,0 +1,120 @@
+# KB(Knowledge Base) Importer
+
+The vulnerability database contains known vulnerabilities for which a fix exists. In fact, the fix (commit fix in the vulnerable library project) of the vulnerability must be provided in order to add a vulnerability to the database.
```

Comment:
Before this, I would add one single sentence that says what KB Importer does (we don't want the reader to waste their time reading the terminology and general explanation if that is not what theyŕe looking for).

Reply:
I have added a small sentence at the top